src/components/logger/index.ts

```diff
@@ -95,9 +95,17 @@ export function probeBuildLog({
       mLog.alert.flag = 'alert'
       mLog.alert.message = ['URI not found']
       break
+    case 1:
+      mLog.alert.flag = 'alert'
+      mLog.alert.message = ['Connection reset']
```

Comment:
Connection reseted

Reply:
haha the past tense of reset is reset.   (english is wierd! :P)